src/app/pages/Swap/form/index.tsx

```diff
@@ -509,171 +537,117 @@ export const SwapForm = memo<Props>(({ publicKeyHash }) => {
   const handleCloseAlert = () => setIsAlertVisible(false);
 
   return (
-    <form className="mb-8" onSubmit={handleSubmit(onSubmit)}>
-      {isAlertVisible && (
-        <Alert
-          closable
-          className="mb-4"
-          type="error"
-          description={<T id="noRoutesFound" />}
-          onClose={handleCloseAlert}
+    <>
+      <form id="swap-form" className="flex flex-col h-full pt-4 flex-grow flex-1" onSubmit={handleSubmit(onSubmit)}>
+        {isAlertVisible && (
+          <div className="px-4">
+            <Alert
+              closable
+              className="flex mb-4"
+              type="error"
+              description={<T id="noRoutesFound" />}
+              onClose={handleCloseAlert}
+            />
+          </div>
+        )}
+
+        {operation && (
+          <div className="px-4">
+            <OperationStatus
+              network={network}
+              className="mb-8"
+              closable
+              typeTitle={t('swapNoun')}
+              operation={operation}
+              onClose={handleOperationClose}
+            />
+          </div>
+        )}
+
+        <SwapFormInput
+          network={network}
+          publicKeyHash={publicKeyHash}
+          className="px-4"
+          name="input"
+          value={inputValue}
+          // eslint-disable-next-line no-type-assertion/no-type-assertion
+          error={errors.input?.message as string}
```

Comment:
```suggestion
          error={errors.input?.message}
```
fix ts after form migration to v7